opm/autodiff/MatrixBlock.hpp

```diff
@@ -206,6 +206,16 @@ static inline void invertMatrix (FieldMatrix<K,n,n> &matrix)
     matrix.invert();
 }
 
+//! invert matrix by calling matrix.invert
+template <typename K>
+static inline void invertMatrix (Dune::DynamicMatrix<K> &matrix)
```

Comment:
Nitpicks:
 - The left parenthesis should be directly adjacent to the function name (as opposed to `for`, `if`, `while` etc.).
 - The `&` should be adjacent to the type, not the parameter name.

Please also fix for the two other invertMatrix() definitions.

Reply:
> The left parenthesis should be directly adjacent to the function name (as opposed to for, if, while etc.).

okay. Actually, I prefer that for all the few situations you mentioned.  But since we `agreed` to use the same style with the neighboring code ... whatever.

Reply:
I misunderstood `parenthesis` and `brackets`. Sorry for the previous noisy message.